### ISSUE TITLE
Support Apple Silicon

### ISSUE THREAD

**Issue by KElee01** [Yang-Xijie/InputSourceSwitcher#9]
Please add support for Apple Silicon.

**Comment by Yang-Xijie** [comment]
I'm not so familiar with M1 Mac and I don't have a M1 machine...
Will it be okay to just change the build target in Xcode?

<img width="336" alt="image" src="https://user-images.githubusercontent.com/57952362/128457043-ef4f6f22-6716-47d0-99ba-580b3f561264.png">

**Comment by Yang-Xijie** [comment]
I'm not sure whether it works.
Could you please try it t see whether it works on Apple Silicon?
[Source Switcher build 6 Apple Silicon.zip](https://github.com/Yang-Xijie/InputSourceSwitcher/files/6943040/Source.Switcher.build.6.Apple.Silicon.zip)

**Comment by KElee01** [comment]
I don't think it works, but it does run natively on arm.
Thank you, it's okay if you are not familiar with it, it's still a nice app.
<img width="429" alt="1" src="https://user-images.githubusercontent.com/54350573/128458450-ede74535-a178-43f6-9e18-795281889fa0.png">
<img width="429" alt="2" src="https://user-images.githubusercontent.com/54350573/128458456-3d2b05a1-4c4e-4d33-b41d-11f6a57771f4.png">

**Comment by Yang-Xijie** [comment]
You mentioned `I don't think it works`.

Did you try clicking the `Reset` button? After resetting, the input sources should appear.

If it still not works, then I have no idea about it...

**Comment by KElee01** [comment]
I don't think it works just because I saw the switch buttons were not shown.
I tried to reset and restart the app but it doesn't work.

**Comment by Yang-Xijie** [comment]
All right. Sorry for that. I'll fix the problem if I have a Mac with Apple Silicon.
By the way, pull requests are welcome!

**Comment by KElee01** [comment]
It's okay, I'm looking forward to it.

**Comment by Yang-Xijie** [comment]
emmm
I think it's better to keep this issue open 😀

**Comment by KElee01** [comment]
Okay..😂

**Comment by n0thing2speak** [comment]
```bash
[2021/11/27, 3:57:21 PM] [Model] createMyInputSources()
[2021/11/27, 3:57:21 PM] [Applescript] NSAppleScript.executeAndReturnError(): {
    NSAppleScriptErrorAppName = "System Events";
    NSAppleScriptErrorBriefMessage = "Source Switcher is not allowed assistive access.";
    NSAppleScriptErrorMessage = "System Events got an error: Source Switcher is not allowed assistive access.";
    NSAppleScriptErrorNumber = "-1719";
    NSAppleScriptErrorRange = "NSRange: {87, 4}";
}
[2021/11/27, 3:57:21 PM] [Applescript] NSAppleScript.executeAndReturnError(): {
    NSAppleScriptErrorAppName = "System Events";
    NSAppleScriptErrorBriefMessage = "Source Switcher is not allowed assistive access.";
    NSAppleScriptErrorMessage = "System Events got an error: Source Switcher is not allowed assistive access.";
    NSAppleScriptErrorNumber = "-1719";
    NSAppleScriptErrorRange = "NSRange: {87, 4}";
}
[2021/11/27, 3:57:22 PM] [NotificationCenter.requestAuthorization] error - Error Domain=UNErrorDomain Code=1 "Notifications are not allowed for this application" UserInfo={NSLocalizedDescription=Notifications are not allowed for this application
```

I did give permission, and it looks like the permission was not given successfully

**Comment by n0thing2speak** [comment]
fix  #16

**Comment by n0thing2speak** [comment]
[Source Switcher.app.zip](https://github.com/Yang-Xijie/InputSourceSwitcher/files/7611494/Source.Switcher.app.zip)

This is the Universal app I compiled for the release version
